monkestation/code/modules/projectiles/projectile/special/rocket.dm

```diff
@@ -34,4 +34,30 @@
 		new/obj/spatial_rift(A.loc)
 		qdel(A)
 
+	if(istype(A, /obj/anomaly/energy_ball))
+		var/mob/living/user = firer
+		user.client.give_award(/datum/award/achievement/misc/singularity_buster, user)
+		user.emote("scream")
+
+		for(var/mob/player as anything in GLOB.player_list)
+			SEND_SOUND(player, 'sound/magic/charge.ogg')
+			to_chat(player, "<span class='boldannounce'>You feel reality distort for a moment...</span>")
+			shake_camera(player, 15, 3)
+
+		new/obj/spatial_rift(A.loc)
+		qdel(A)
+
+	if(istype(A, /obj/eldritch/narsie))
```

Comment:
And this can just be /obj/eldritch to catch all possibilities

Reply:
Just made this one